relayer/yarn.lock

```diff
@@ -432,11 +432,11 @@
     snarkjs "^0.7.0"
 
 "@lightprotocol/zk.js@file:../light-zk.js":
-  version "0.3.2-alpha.13"
+  version "0.3.2-alpha.14"
   dependencies:
     "@coral-xyz/anchor" "0.28.0"
     "@coral-xyz/borsh" "^0.28.0"
-    "@lightprotocol/prover.js" "file:../../../.cache/yarn/v6/npm-@lightprotocol-zk-js-0.3.2-alpha.13-1d566ce6-be9f-4af7-a6c7-fbafc28a9dcd-1693341855616/node_modules/@lightprotocol/light-prover.js"
+    "@lightprotocol/prover.js" "file:../../../Library/Caches/Yarn/v6/npm-@lightprotocol-zk-js-0.3.2-alpha.14-7dee7c01-211c-48cf-b05d-7f5cb40975b3-1694707767038/node_modules/@lightprotocol/light-prover.js"
```

Comment:
This looks very bad. Of course that diff is because of difference between Linux (`.cache/yarn`) and macOS (`Library/...`), We need to check if there is any other reason for this happening than just having prover.js as a `file:` dependency.

And if that's only because of using it as `file:` dependency, we really need to specify all deps with releases, always, and do `yarn link` for local development.

Reply:
agreed - tho that's for a different pr no?

Reply:
also cc @ananas-block so we can log this issue

Reply:
looks ugly. This will go away with the pr which removes the relative path imports.